docs/pages/client/admin.mdx

```diff
@@ -39,6 +39,17 @@ import { Callout } from 'nextra-theme-docs'
 
 - Create, edit, and delete your various tileservers
 - Tileservers are used for selecting different map tiles from the Kōji client
+- Currently only tested with: [123FLO321/SwiftTileserverCache](https://github.com/123FLO321/SwiftTileserverCache)
```

Comment:
It also works with the standard remote ones. OSM, Satellite, Voyager, etc. Not sure if it's worth mentioning alongside the custom examples?

Reply:
Agreed. I'm not including all common servers in this example but at least include the Default that Koji ships with.